restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -195,10 +196,10 @@
 	"externalServiceStepChooseTargetLabel": "Seleccione módulo de destino",
 	
 	"faxStepFaxLabel": "Contenido fax",
-	"faxStepFaxLabelPopover": "Escriba el contenido fax como texto. de Type the fax content as text. Utilice las variables con el signo de dollar como '$variablename'",
+	"faxStepFaxLabelPopover": "Escriba el contenido fax como texto. Utilice las variables con el signo de dollar como '$variablename'",

```

Comment:
Grammar error found. Change for:
"faxStepFaxLabelPopover": "Escriba el contenido fax como texto. Utilice las variables con el signo de dólar como '$variablename'",